packages/dds/test-dds-utils/src/ddsFuzzHarness.ts

```diff
@@ -43,6 +44,8 @@ import {
 	MockFluidDataStoreRuntime,
 	MockStorage,
 } from "@fluidframework/test-runtime-utils";
+import { v4 as uuid } from "uuid";
```

Comment:
Is there somewhere else I should import this from?

Reply:
should be fine. i don't know why you need the case below